tastyworks/models/order.py

```diff
@@ -94,15 +94,52 @@ def add_leg(self, security: Security):
     @classmethod
     def from_dict(cls, input_dict: dict):
         """
-        Parses an Order object from a dict.
+        Parses an Order object from an order dict.
+
+        ...as of 4/17/2021...
+        {
+        'id': abc,
+        'account-number':'abc123',
+        'time-in-force': 'GTC',
+        'order-type': 'Limit',
+        'size': 1,
+        'underlying-symbol': 'SPY',
+        'underlying-instrument-type': 'Equity',
+        'price': '0.94',
+        'price-effect': 'Debit',
+        'status': 'Received',
+        'cancellable': True,
+        'editable': True,
+        'edited': False,
+        'ext-exchange-order-number': '123',
+        'ext-client-order-id': '123',
+        'ext-global-order-number': 123,
+        'received-at': '2021-04-06T17:06:38.904+00:00',
+        'updated-at': 1618604409279,
+        'legs': [{
+                'instrument-type': 'Equity Option',
+                'symbol': 'SPY   210716P00320000',
+                'quantity': 1,
+                'remaining-quantity': 1,
+                'action': 'Buy to Close',
+                'fills': []
+                }]
+        }
```

Comment:
What is the purpose of having this here? Should we not put it either as a comment (so that it's not in the dosctring) or move it to a separate folder with these examples documented?

Something along the lines of `/docs/api/responses/4_april_2021/name_of_endpoint.txt`?

Reply:
Yes, sorry, I will make a better documentation folder with all the data I collected, that way we could have a logical place for all these comments.